dist/chart/templates/manager/manager.yaml

```diff
@@ -69,7 +71,6 @@ spec:
       securityContext:
         {{- toYaml .Values.controllerManager.securityContext | nindent 8 }}
       serviceAccountName: {{ .Values.controllerManager.serviceAccountName }}
-      hostNetwork: {{ .Values.controllerManager.hostNetwork }}
```

Comment:
This needs to stay.

Reply:
Kubebuilder doesn't generate this field. However, if you want the `hostNetwork` field to stay, I will have to add it manually. Should I do it manually?

Reply:
added it back